src/components/memorygame/MemoryGameHowToPlayPopover/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { QuestionCircleTwoTone } from "@ant-design/icons";
+import { useTranslation } from "react-i18next";
+import { Popover } from "antd";
+import "./index.css";
+
+const HowToPlayPopover = () => {
+  const [t] = useTranslation();
+  const howToPlayTitle = t("memoryGame.howToPlayTitle");
```

Comment:
same comment here as i put in your other PRs. can you fix in all your PRs

Reply:
I will

Reply:
PTAL

Reply:
are organizing the translation in the locales as you did it or like this:
`t("games.memoryGame.howToPlayTitle")`

Reply:
We didn't agree on it as group. I realized someone is using that format and decided to follow it.

Reply:
maybe we can discuss today because I also followed someone else's work and now it's confusing